environments/puppet/hiera.yaml

```diff
@@ -6,6 +6,21 @@ defaults:
 
 hierarchy:
 
+  - name: Eyaml Data
+    lookup_key: eyaml_lookup_key
+    paths:
+    - "secrets/hosts/%{trusted.certname}.eyaml"
+    - "secrets/hosts/%{facts.fqdn}.eyaml"
+    - "secrets/hosts/%{facts.hostname}.eyaml"
+    - "secrets/domains/%{facts.domain}.eyaml"
+    - "secrets/hostgroups/%{::hostgroup}.eyaml"
+    - "secrets/common.eyaml"
```

Comment:
We don't really _need_ to add a new `secrets/` tree; we could have .yaml and .eyaml files sitting side-by-side in the same hierarchy.

Reply:
Made changes.